### PR TITLE
Fixes for How To Write How To Guides

### PR DESCRIPTION
**Pull request by cmelliott2**
### Description

Minor language and formatting changes in How to Write a MoveIt How-To Guide

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

### CODE REVIEW

File: doc/how_to_contribute/how_to_write_how_to_guides.rst [hunk 1]
```diff
@@ -37,15 +37,15 @@ Steps
 
    - If you think people may regularly find this particular guide by mistake, add links to the proper resource.
 
-#. Write the specific learning objectives (i.e. what will the reader know when they finish reading this how-to).
+#. Write the specific learning objectives (i.e. what the reader will know when they finish reading this how-to).
```
Comment (jackcenter) [inline doc/how_to_contribute/how_to_write_how_to_guides.rst:40]:
```suggestion
#. Write the specific learning objectives (i.e., what the reader will know when they finish reading this how-to).
```

File: doc/how_to_contribute/how_to_write_how_to_guides.rst [hunk 2]
```diff
@@ -55,29 +55,29 @@ Template
...
   ---------------
 
-  <A list of links to related content on and off this website>
+  <A list of links to related content on and off this website.>
 
 Further Reading
 ---------------
 - :doc:`/doc/how_to_contribute/how_to_contribute_to_site`
-- Concepts page: :doc:`/doc/concepts/how_to_guide`
+- :doc:`MoveIt Concepts: Tutorials </doc/concepts/tutorials>`
```
Comment (jackcenter) [inline doc/how_to_contribute/how_to_write_how_to_guides.rst:83]:
```suggestion
- :doc:`MoveIt Concepts: How-To Guide </doc/concepts/how_to_guide>`
```

**Review by mlautman — COMMENTED** [review]
Looks good to me but I don't have a linux machine build your changes locally. If someone else verifies that the syntax for links/formatting is correct, I approve

File: doc/how_to_contribute/how_to_write_how_to_guides.rst [hunk 3]
```diff
@@ -14,20 +14,20 @@ Requirements
...
-- MoveIt2
+- MoveIt 2
 
 Steps
 -----
 
-1. Fork the MoveIt2 Tutorials repository (https://github.com/ros-planning/moveit2_tutorials.git) and start a new branch with an intuitive name (e.g. jack/how-to-write-how-tos)
+1. Fork the `MoveIt 2 Tutorials repository <https://github.com/ros-planning/moveit2_tutorials.git>`_ and start a new branch with an intuitive name (e.g., ``jack/how-to-tutorials``).
 
 #. Create a new file in the ``doc/how_to_guides`` directory with a ``.rst`` extension. The title should start with "How to" followed by the specific question being answered (e.g. "How to Write a MoveIt How-To Guide").
```
Comment (corycrean) [inline doc/how_to_contribute/how_to_write_how_to_guides.rst:24]:
```suggestion
#. Create a new file in the ``doc/how_to_guides`` directory with a ``.rst`` extension. The title should start with "How to" followed by the specific question being answered (e.g., "How to Write a MoveIt How-To Guide").
```

File: doc/how_to_contribute/how_to_write_how_to_guides.rst (hunk 1 above)
Comment (corycrean) [inline doc/how_to_contribute/how_to_write_how_to_guides.rst:40]:
```suggestion
#. Write the specific learning objectives (i.e., what the reader will know when they finish reading this how-to).
```

File: doc/how_to_contribute/how_to_write_how_to_guides.rst [hunk 4]
```diff
@@ -55,29 +55,29 @@ Template
...
   -------------------
 
-  <List of things the user will learn>
+  <List of things the user will learn.>
 
   Requirements
   ----------
 
-  <Explination of what the user should understand.  Unlike a tutorial these are stand-alone and can assume the user has much more background.>
+  <Explanation of what the user should understand. Unlike a tutorial these are stand-alone and can assume the user has much more background.>
```
Comment (corycrean) [inline doc/how_to_contribute/how_to_write_how_to_guides.rst:68]:
```suggestion
  <Explanation of what the user should understand. Unlike a tutorial, these stand alone and can assume the user has much more background.>
```

**Review by corycrean — CHANGES_REQUESTED** [review]

File: doc/how_to_contribute/how_to_write_how_to_guides.rst [hunk 5]
```diff
@@ -14,20 +14,20 @@ Requirements
...
 - Ubuntu 20.04
 - ROS 2 Galactic
-- MoveIt2
+- MoveIt 2
 
 Steps
 -----
 
-1. Fork the MoveIt2 Tutorials repository (https://github.com/ros-planning/moveit2_tutorials.git) and start a new branch with an intuitive name (e.g. jack/how-to-write-how-tos)
+1. Fork the `MoveIt 2 Tutorials repository <https://github.com/ros-planning/moveit2_tutorials.git>`_ and start a new branch with an intuitive name (e.g., ``jack/how-to-tutorials``).
```
Comment (cmelliott2) [inline doc/how_to_contribute/how_to_write_how_to_guides.rst:22]:
```suggestion
1. Fork the `MoveIt 2 Tutorials repository <https://github.com/ros-planning/moveit2_tutorials.git>`_ and start a new branch with an intuitive name (e.g., ``jack/how-to-write-how-tos``).
```

File: doc/how_to_contribute/how_to_write_how_to_guides.rst (hunk 4 above)
Comment (cmelliott2) [inline doc/how_to_contribute/how_to_write_how_to_guides.rst:68]:
```suggestion
  <Explanation of what the user should understand. Unlike a tutorial, these guides stand alone and can assume the user has much more background.>
```

**Review by corycrean — APPROVED** [review]

**Review by tylerjw — APPROVED** [review]

**Pull request merged**